debezium-core/src/main/java/io/debezium/data/geometry/Point.java

```diff
@@ -6,72 +6,125 @@
 
 package io.debezium.data.geometry;
 
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.apache.kafka.connect.data.Struct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A semantic type for a geometric Point, defined as a set of (x,y) coordinates.
- * 
+ *
+ * This historically used to be a useful class, but is now a MySQL-only type to be deprecated.
```

Comment:
This used to be for PG, too, and IIUC, it still is, with this PR.

I'd also be curious about the thoughts of @omarsmak on this, as he did this initial work. I'd suppose he wouldn't like the removal of those dedicated coords fields too much.

Reply:
Hi @gunnarmorling , thank you for inviting me. Initially we did this work since the specific GEO coordinate were important to us along with the Well-known binary and we agreed to have a specific Geo types (POINT..etc). However, I can see the point that @rcoup raised here which you'd rely more on deserialising WKB downstream using a geometry processing library to manipulate the data, which is true (At least for us now). However, the question whether we want the coords fields or not and just rely on WKB, I will have to discuss this internally with my team and have a common understanding to just rely on WKB for our downstream sinks implementation, does that make sense @gunnarmorling @rcoup ?

Reply:
To be clear the existing PostgreSQL Point type still works as-is — but this is/was only the weird built-in/legacy Point type. PostGIS geometries were never supported before, and are now implemented as a Geometry. (In PostgreSQL they're completely separate, and so now debezium treats them as such).

For MySQL the existing Point type is still present with the X & Y attributes _as well as_ the WKB/SRID. WKB obviously supports 2D+M & 3D points too.